README.md

```diff
@@ -1,30 +1,124 @@
+local language folder, e.g. [Localization/zh-CN](Localization/zh-CN), update
+the metadata field.  For each file, translate each ``msgid`` to ``msgstr``. 
+
+We are using Markdown for formatting the strings to be translated and the 
+translated string. We have made the decision to support the following Markdown
+syntaxes: 
+
+ - Links: e.g. ``[Example](http://example.net/)``
+ - Bold text: e.g. ``**Example**``
+ - Italic: e.g. ``*example*``
+
+For more information on Markdown, feel free to have a look at 
+[Github's official guide](https://guides.github.com/features/mastering-markdown/),
+ however, we are only supporting the formatting syntaxes statated above.
+
+You can use a text editor, such as [Notepad++](https://notepad-plus-plus.org/) 
+or [Atom](https://atom.io/). Please **Set your encoding to UTF-8, this is 
+important!**. Alternatively you can use tools such as 
+[Poedit](https://poedit.net/) or 
+[Lokalize](https://kde.org/applications/en/office/org.kde.lokalize). We do not
+mind the extra metadata these tools add. 
+
+### PO file mandatory metadata
+At the top of each PO file, it is mandatory to have the following metadata
+fields:
+
+ 1. ``Schema-Version`` - The schema version of this PO file, e.g. ``1.2``.
+ In future, we may have minor update to the format of the PO files.
+ 2. ``URL`` - The URL of the webpage that generated the PO file.
+ 3. ``Source-Language`` - The original language of the webpage, it should be
+ ``en-US`. However, as FAH project grows, we may start seeing webpages that were
+ written in language that was not English.
+ 4. ``Source-Version`` - The version code for the source PO file,
+  e.g. ``1.2``.
+ 5. ``Source-Last-Modified`` - The last modification date for the source PO
+ file, this must be  in the format of **YYYYMMDD**, e.g. ``20200704``.
+ 6. ``Target-Language`` - The target language for this PO file, e.g. ``zh-CN``.
+ 7. ``Target-Version`` - The version code for the target PO file,
+ e.g. ``1.2``.
+ 8. ``Target-Last-Modified`` - The last modification date for this PO file, this
+ must be in the format of **YYYYMMDD**, ``20200704``.
+
+Please note that: 
+
+ - For dates, ``YYYY`` is the 4-digit year, ``MM`` is the 2-digit month,
+ and ``DD`` is the 2-digit day.
+ - The ``Source-Version`` should follow the format of ``x.y``, where ``x`` and
+ ``y`` are both integers. Increase in ``x`` indicates a major structure
+ change, e.g. adding or removing a paragraph. Increase in ``y`` indicates a
+  minor change, e.g. a spelling mistake. Note that ``x`` and ``y`` can have
+  multiple digits, e.g. ``1.12``. ``y`` should start at 0.
+ - The ``Target-Version`` should also follow the format of ``x.y.z``. ``x`` and
+ ``y`` should follow the ``x`` and ``y`` in ``Original-Version``. ``z`` should
+ start at 0. It should be incremented before pushing the updated PO file to the
+ main repository. ``z`` should be reset to 0, if a new ``Source-Version`` is
+ released.
```

Comment:
Do we really need this explanation? Can we not just say "please stick to the formats used in the original files"? Seems a bit overwhelming to me.

Reply:
I think it is better to leave them here, just for future reference. It is useful to know the decision behind the design choice.

We can have a TL;DR version / a shorter version for volunteers.